F/Fmt/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Fmt"
+version = v"9.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/fmtlib/fmt.git", "a33701196adfad74917046096bf5a2aa0ab0bb50")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+INSTALL_PATH=${prefix}
+cd $WORKSPACE/srcdir/fmt
+mkdir build
+cd build
+cmake \
+-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+-DBUILD_SHARED_LIBS=True \
+-DCMAKE_INSTALL_PREFIX=${INSTALL_PATH} \
+..
+make
+make install
+
+# the license file isn't found automatically for some reason
+# perhaps it's the .rst extension?
+#
+mkdir ${prefix}/share
+mkdir ${prefix}/share/licenses
+mkdir ${prefix}/share/licenses/Fmt
+cp $WORKSPACE/srcdir/fmt/LICENSE.rst ${prefix}/share/licenses/Fmt/LICENSE
```

Comment:
You might really want to [read the documentation](https://docs.binarybuilder.org/stable/build_tips/#Installing-the-license-file)
```suggestion
install_license ../LICENSE.rst
```